src/com/owncloud/android/providers/FileContentProvider.java

```diff
@@ -96,20 +96,25 @@ private int delete(SQLiteDatabase db, Uri uri, String where, String[] whereArgs)
         switch (mUriMatcher.match(uri)) {
             case SINGLE_FILE:
                 Cursor c = query(db, uri, null, where, whereArgs, null);
```

Comment:
If we are going to add a ```catch``` block for runtime exceptions, then we should include this line in the ```try``` block, separating the declaration of ```c``` from its assignment, like:

```
Cursor c= null;
try {
   c= query(db, uri, null, where, whereArgs, null);
```